corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/util.js

```diff
@@ -121,6 +121,22 @@ hqDefine("cloudcare/js/formplayer/menus/util", function () {
         } else if (menuResponse.type === "query") {
             return hqImport("cloudcare/js/formplayer/menus/views/query")(menuData);
         } else if (menuResponse.type === "entities") {
+            if (hqImport('hqwebapp/js/toggles').toggleEnabled('APP_ANALYTICS')) {
+                var Util = hqImport("cloudcare/js/formplayer/utils/util");
+                var urlObject = Util.currentUrlToObject();
+                var searhText = urlObject.search;
+                var event = "Viewed Case List";
+                var eventData = menuResponse.title;
+                if (searhText) {
+                    event = "Searched Case List";
+                    eventData = searhText;
```

Comment:
I think this wants be the title still. We don't want the search text itself because it might contain private info, but we do want to know which case list they searched.